pom.xml

```diff
@@ -214,7 +214,7 @@
     <maven-antrun.version>3.1.0</maven-antrun.version>
     <commons-crypto.version>1.1.0</commons-crypto.version>
     <commons-cli.version>1.5.0</commons-cli.version>
-    <bouncycastle.version>1.60</bouncycastle.version>
+    <bouncycastle.version>1.70</bouncycastle.version>
```

Comment:
OK I guess this doesn't show up in the deps/ files? OK

Reply:
Yes, just a test dependency, only used by Yarn MiniYARNCluster